app/models/exported_plan.rb

```diff
@@ -99,24 +99,40 @@ def title
 
   # Export formats
 
-  def as_csv
+  def as_csv(sections, unanswered_questions, question_headings)
     CSV.generate do |csv|
-      csv << [_('Section'),_('Question'),_('Answer'),_('Selected option(s)'),_('Answered by'),_('Answered at')]
-      self.sections.each do |section|
-        questions = self.questions_for_section(section)
-        if questions.present?
-          questions.each do |question|
-            answer = self.plan.answer(question.id)
-            q_format = question.question_format
-            if q_format.option_based?
-              options_string = answer.question_options.collect {|o| o.text}.join('; ')
-            else
-              options_string = ''
-            end
+      if question_headings
```

Comment:
Not convinced at all about CSV format's usefulness at this stage - we've had fewer than 240 downloads in total and I'm pretty sure more than half of those are us testing it...